cmd/osbuild-composer-cli-tests/main_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
Is the `// +build integration` below it not sufficient?

Reply:
Interesting. I didn't add that on purpose. I'm guessing vim-go may have done that...

Reply:
Ah, I see. go:build is now preferred by go 1.18 and, so I'll remove the old form.